eth/stagedsync/stage_snapshots.go

```diff
@@ -477,6 +483,8 @@ func SnapshotsPrune(s *PruneState, initialCycle bool, cfg SnapshotsCfg, ctx cont
 				}
 
 				return nil
+			}, func() error {
+				return pruneBlockSnapshots(ctx, cfg, logger)
```

Comment:
let's add next logic: if it deleted any file - it need return `true` and we need call 
```
if notifier != nil {
   notifier.OnNewSnapshot()
}
```